typescript-tsx-mode/componentTSX

```diff
@@ -0,0 +1,16 @@
+# -*- mode: snippet -*-
+# contributor: Talha Abid 
+# name: componentTSX
+# key: componentTSX
+# --
+import React from 'react';
+
+interface ${1:`(f-base buffer-file-name)`}PropTypes {
```

Comment:
Use `file-name-base` instead of `f-base`. `f` is not one of `doom-snippets` and I want to avoid making it one.

Reply:
Alright done but can I ask what key does since I was following the old examples in the code base , is key what I will type to expand the snippet ?

Reply:
Exactly, yeah.﻿ `fet` + <kbd>tab</kbd> will expand the snippet.